### DISCUSSION TITLE
Wyze light switch integration?

### DISCUSSION THREAD

**Discussion by jastsai306** [SecKatie/ha-wyzeapi#299] [Ideas]
Wyze recently introduced their light switch. Do you guys plans on adding this integration? Thank you.

**Comment by SecKatie** [comment]
I do not have one but I think there is an open issue you can follow. If we get the data I think @yoinx was planning on putting some code together for it

**Reply by eschen66** [reply]
Any update? Nothing new in the open issue section. Would be very nice if it was added and I’d probably fill my house with them. Currently using them by creating an Alexa “play media” button in HA. This method is not optimal though since I can’t view state and have to have a button for “turn X light on” and “turn X light off”. Thanks!

**Reply by SecKatie** [reply]
I am not working on this. If no one else is willing to contribute the code for this I doubt we will see it anytime soon.

**Reply by eschen66** [reply]
I’ll try to figure it out when I get home today and see if I can figure out some code for it.

**Reply by Flash3200** [reply]
Funny, today I setup an automation for the Amazon echo in my office to tell the google home mini to turn on the switch!  Would love to get this added!

**Comment by danielzrob** [comment]
I ended up using ifttt and webhooks
